build/bakefiles/common.bkl

```diff
@@ -37,6 +37,8 @@
         <if cond="FORMAT=='msvs2008prj' and MSVS_PLATFORM=='win64'">_x64</if>
         <if cond="TARGET_CPU=='amd64'">_x64</if>
         <if cond="TARGET_CPU=='AMD64'">_x64</if>
+        <if cond="TARGET_CPU=='arm'">_arm64</if>
+        <if cond="TARGET_CPU=='ARM'">_arm64</if>
```

Comment:
I'm almost sure that we shouldn't have `64` here, should we?

Reply:
That's right, thanks!